tests/gold_tests/timeout/tls_conn_timeout.test.py

```diff
@@ -45,60 +49,68 @@
 ts.Disk.remap_config.AddLine('map /get_connect_blocked https://127.0.0.1:{0}'.format(Test.Variables.get_block_connect_port))
 ts.Disk.remap_config.AddLine('map /get_ttfb_blocked https://127.0.0.1:{0}'.format(Test.Variables.get_block_ttfb_port))
 
+# Commenting out the per test case timeouts.  In the CI, there is too big of a risk that we hit those timeouts.  Had hoped to use
+# The test case timeouts to differentiate between a good origin timeout and a too long origin timeout
+
 # Run the connection and TTFB timeout tests first with POST.
 
 # Request the port that should timeout in the handshake
 # Should retry once
-tr = Test.AddTestRun("tr-blocking")
-tr.Setup.Copy("case2.sh")
-tr.Processes.Default.StartBefore(Test.Processes.ts, ready=When.PortOpen(ts.Variables.port))
+tr = Test.AddTestRun("tr-blocking-post")
 tr.Setup.Copy("../chunked_encoding/ssl/server.pem")
-tr.Processes.Default.Command = 'sh ./case2.sh {0} 3 0 {1} connect_blocked'.format(Test.Variables.block_connect_port, ts.Variables.port)
-tr.Processes.Default.TimeOut = 6
+tr.Processes.Default.StartBefore(Test.Processes.ts, ready=When.PortOpen(ts.Variables.port))
```

Comment:
We shouldn't need this ready condition. I actually thought I removed all of these. The default MakeATSProcess function should set the ready that waits upon a log message from traffic server saying that it is ready to optimize traffic.

Can you please remove this ready condition from the patch?

Reply:
Addressed.